src/Params/ADnoteParameters.cpp

```diff
@@ -720,6 +726,7 @@ ADnoteGlobalParam::~ADnoteGlobalParam()
     delete FilterEnvelope;
     delete FilterLfo;
     delete Reson;
+    delete wskernel;
```

Comment:
Valgrind showed an error here. The `new[]` must be matched with `delete[]`, not with `delete`.
```suggestion
    delete [] wskernel;
```